source/MaterialXView/Viewer.cpp

```diff
@@ -247,7 +247,9 @@ Viewer::Viewer(const std::string& materialFilename,
     _bakeHdr(false),
     _bakeAverage(false),
     _bakeOptimize(true),
-    _bakeRequested(false)
+    _bakeRequested(false),
+    _bakeWidth(0),
```

Comment:
Although the user can now trigger a texture bake from the command line, it doesn't look like the specified values of `_bakeWidth` and `_bakeHeight` are yet respected by the Viewer::bakeTextures method.  If you still need support for manual baking resolutions (as opposed to always using the computed resolution), make sure you check these two values in Viewer::bakeTextures and pass them along to the constructed baker.  If computed resolutions are sufficient for your workflows, then you could instead remove the bakeWidth and bakeHeight options.

Reply:
I still wish to limit the textures. Will look into the error. It takes time for me to work in a new code base.

Reply:
QA testing.

Reply:
```
 .\MaterialXView.exe --bakeWidth 8912 --bakeHeight 8192 --material C:\Users\ernest.lee\Downloads\MaterialX_Windows_VS2019_x64_Python38\resources\Materials\Examples\StandardSurface\standard_surface_brass_tiled.mtlx --bakeFilename bake_02.mtlx
```